src/tool_operate.c

```diff
@@ -455,6 +455,14 @@ static CURLcode post_per_transfer(struct GlobalConfig *global,
       if(ECONNREFUSED == oserrno)
         retry = RETRY_CONNREFUSED;
     }
+    else if (CURLE_FILE_COULDNT_READ_FILE == result) {
+      /*
+       * treat unreadable file condition as retryable and regard it as
+       * `refused' in a way, so that the requested file may be
+       * available while retrying (= waiting for some processing)
+       */
+      retry = RETRY_CONNREFUSED;
```

Comment:
... Or, instead of abusing `RETRY_CONNREFUSED`, this condition may be granted to give its own enum for the sake of more proper messages.